app/(tabs)/movie/[id]/tickets.tsx

```diff
@@ -14,6 +16,9 @@ export default function TicketsScreen() {
   const showtimeId = bookingData?.selectedShowtimeId || null
 
   const { data: ticketsData, isLoading, error } = useShowtimeTickets(showtimeId)
+  const { data: showtimeData } = useShowtime(showtimeId)
+
+  const ticketPrice = showtimeData?.ticket_price
```

Comment:
I think in the get user tickets endpoint you added the ticket price, so you shouldn't need this